trello_items.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import secrets
+from card import Card
+
+key = secrets.apiKey
+token = secrets.apiToken
+
+def get_todo_cards_for_board():
+    """
+    Fetches all To Do cards from the board.
+
+    Returns:
+        list: The list of cards.
+    """
+
+    query = {
+        'key': key,
+        'token': token
+    }
+   
+    board_id = secrets.boardId
+    to_do_cards = []
+    url = "https://api.trello.com/1/boards/{}/cards".format(board_id)
```

Comment:
If you have an up to date version of python you should be able to write `f"https://api.trello.com/1/boards/{board_id}/cards"` which is easier to read.

Reply:
Updated as I have a new version of Python